docs/en/observability/monitor-azure-agent.asciidoc

```diff
@@ -0,0 +1,382 @@
+value after you leave this page!
+====
+
+[discrete]
+==== Grant access permission for your service principal
+
+After creating the Azure service principal, you need to grant it the correct
+permission. You need `Reader` permission to configure {agent} to monitor
+your services.
+
+. In the Azure Portal, search for and select **Subscriptions**.
+. In the Subscriptions page, click the name of your subscription.
+. In the navigation pane, select **Access control (IAM)**.
+. Click **Add** and select **Add role assignment**.
+. On the **Roles** tab, select the **Reader** role, then click **Next**.
+. On the **Members** tab, select the option to assign access to
+**User, group, or service principal**.
+. Click **Select members**, then search for and select the principal you created
+earlier.
+. For the description, enter the name of your service principal.
+. Click **Next** to review the role assignment:
++
+[role="screenshot"]
+image::images/agent-tut-azure-add-role-assignment.png[Screen capture of adding a role assignment]
+
+. Click **Review + assign** to grant the service principal access to your
+subscription.
+
+[discrete]
+[[elastic-agent-add-azure-integration]]
+=== Step 2: Install the Azure Billing Metrics integration
+
+In this step, you install the Azure Billing Metrics integration in {kib}. This
+integration contains an input for collecting metrics, such as Azure usage
+details and forecast information, about your subscription.
+
+To add the integration: 
+
+. Go to the {kib} home page and click **Add integrations**.
++
+[role="screenshot"]
+image::images/kibana-home.png[Screenshot of the {kib} home page]
+
+. In the query bar, search for **Azure Billing** and select the Azure Billing
+Metrics integration to see more details about it.
+
+. Click **Add Azure Billing Metrics**.
+
+. Under Integration settings, configure the integration name and optionally add
+a description.
++
+TIP: If you don't see options for configuring the integration, you're probably
+in a workflow designed for new deployments. Follow the steps, then return to
+this tutorial when you're ready to configure the integration.
+
+. Specify values for all the required fields. For more information about these
+settings, refer to the {integrations-docs}/azure_billing[Azure Billing Metrics]
+documentation.
++
+--
+[horizontal]
+**Client ID**::
+The Application (client) ID that you copied earlier when you created the service
+principal. 
+
+**Client secret**:: The secret value that you copied earlier.
+
+**Tenant ID**:: The tenant ID listed on the main Azure Active Directory Page.
+
+**Subscription ID**:: The subscription ID listed on the main Subscriptions page.
+--
++
+[role="screenshot"]
+image::images/agent-tut-azure-integration-settings.png[Screenshot of integration settings for Azure]
+
+. Make sure the **Collect Azure Billing metrics** selector is turned on, and
+optionally change the default period to a multiple of 24h.
```

Comment:
The 24h period should be threaded like an advanced setting. The integration makes some assumptions about the period, so users should not change this value unless they know what they are doing.

Reply:
Sounds good. I won't mention this setting in the tutorial.